static/js/pages/receipts.js

```diff
@@ -16,7 +16,7 @@ var columns = [
     className: 'all',
     width: '200px',
     render: function(data, type, row, meta) {
-      if (data) {
+      if (data && row.receipt_type !== '15E') {
```

Comment:
We don't generally reference these types of codes in the frontend code, so what would you think about setting a constant like `EARMARKED = '15E'` to make this check a little clearer?